lib/child_process.js

```diff
@@ -466,8 +466,10 @@ function normalizeSpawnArguments(file, args, options) {
   var env = options.env || process.env;
   var envPairs = [];
 
-  for (var key in env) {
-    envPairs.push(key + '=' + env[key]);
+  for (const [key, value] of Object.entries(env)) {
```

Comment:
I am not sure how fast `Object.entries` is and I somewhat expect it to be slower than e.g. using `Object.keys` or using `for ... in` with a `Object.hasOwnProperty.call(env, key)` check. The latter will definitely be faster in upcoming v8 versions but probably not right now.